pytest-embedded-idf/pytest_embedded_idf/dut.py

```diff
@@ -127,3 +129,88 @@ def close(self) -> None:
         if not self.skip_check_coredump:
             self._check_coredump()
         super().close()
+
+    Case = namedtuple('Case', 'index name type keyword group attribute subcase')
+
+    def get_test_menu(
```

Comment:
`get_xxx` is usually a simple get function. IMHO it would be better if the function is named `parse_test_menu`